### PR TITLE
Open browser version upgrade

### PR DESCRIPTION
**Pull request by tphuong**
Please help to review

### CODE REVIEW

File: AK20/src/test/java/Ex2Upgrade/Ex2UpgradeMain.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        Thread.sleep(2000);
+        driver.quit();
+    }
+
+    public static WebElement getElement(How how, String locator){
+        return driver.findElement(how.buildBy(locator));
+    }
+
+    public static void login(How how, String locator){
+        driver.findElement(how.buildBy(locator)).submit();
```
Comment (hanhtien02) [inline AK20/src/test/java/Ex2Upgrade/Ex2UpgradeMain.java:35]: You can use getElement menthod here.
Ex. driver.getElement(how, locator).submit();

File: AK20/src/test/java/Ex2Upgrade/Ex2UpgradeMain.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+public class Ex2UpgradeMain {
+    public static WebDriver driver;
+    public static void main (String []args) throws InterruptedException {
+
+        openBrowser("Chrome");
+        visitWebsite("https://the-internet.herokuapp.com/login");
+
+        fill(getElement(How.NAME,"username"),"tomsmith");
+        fill(getElement(How.NAME,"password"),"SuperSecretPassword!");
```
Comment (hanhtien02) [inline AK20/src/test/java/Ex2Upgrade/Ex2UpgradeMain.java:20]: You can call getElement inside fill method and just input 3 variables ( how, locator, inputText). 
And your main will be more similar to a test case ( almost non-technical syntax).
Regards,

File: AK20/src/test/java/Ex2Upgrade/Ex2UpgradeMain.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    public static WebElement getElement(How how, String locator){
+        return driver.findElement(how.buildBy(locator));
+    }
+
+    public static void login(How how, String locator){
+        driver.findElement(how.buildBy(locator)).submit();
+    }
+
+    public static void logout(How how, String locator){
+        driver.findElement(how.buildBy(locator)).click();
```
Comment (hanhtien02) [inline AK20/src/test/java/Ex2Upgrade/Ex2UpgradeMain.java:39]: Same idea here: getElement method.

Please let me know if you have better idea.

**Review by vinhlt1988 — APPROVED** [review]

File: AK20/src/test/java/Inspect/LoginPageTest.java [hunk 4]
```diff
@@ -22,6 +22,7 @@ public static void main(String[] args) {
         driver.findElement(By.cssSelector("a[href='/logout']")).submit();
         //todo: find the solution to use switch with string variable
         String name;
+        //test test
```
Comment (hanhtien02) [inline AK20/src/test/java/Inspect/LoginPageTest.java:25]: tien test

**Review by hanhtien02 — APPROVED** [review]
thank you :d

File: AK20/src/test/java/Ex2Upgrade/Ex2UpgradeMain.java (hunk 3 above)
Comment (tphuong) [reply AK20/src/test/java/Ex2Upgrade/Ex2UpgradeMain.java:39]: @hanhtien02 thanks for the detailed reviewing ;) actually I'm just practicing with different methods

**Pull request merged**